### DISCUSSION TITLE
Infiltration 2.9 MOD: The Procedure entry point ... WWindow ... could not be located ...

### DISCUSSION THREAD

**Discussion by Odie3** [OldUnreal/UnrealTournamentPatches#1553] [General] [answered]
I thought I'd try this patch but the Infiltration Mod isn't happy with it all (both client and direct server).  Here is the popup I get when I try to run Infiltration.exe

Or is it no way to patch up when using a UT Mod like Infiltration?

![2024-03-23_12-24-33](https://github.com/OldUnreal/UnrealTournamentPatches/assets/164665741/9c13280f-f1d2-4d0d-8388-29de4ce952e9)

**Comment by SeriousBuggie** [comment]
You need use UnrealTournament.exe not custom one. just pass here your mod INI files and all most work.
`-INI=` and `-USERINI=` is what you need.

**Comment by Odie3** [comment]
Thank's for the quick answer.  I am getting the same error when I run ucc.exe (direct server) for Infiltration. Which is something about the Wheel Mouse :( 

![2024-03-23_12-53-41](https://github.com/OldUnreal/UnrealTournamentPatches/assets/164665741/9a70890f-3aea-462f-87b2-df30c82de42f)

**Comment by SeriousBuggie** [comment]
Did you replace all files from patch? It look as some files keep old.

**Comment by Odie3** [comment]
Thanks!

I had replaced all the UT99 files, I did not replace anything in the Infiltration Mod's folder - which has its own UT99 like setup - ie has a "System" folder.

So I figured I'd updated just the ../Infiltraion/System/ folder too and that allowed it to work when using the ../System/UnrealTournament.exe in my Infiltration shortcut.  Though it makes me sad that the start up Infiltration the logo has been replace (reverted) the UT99 logo.  :'(

I also had to update my UT99 451 server, since it started to send UMenu and other UT99 packages and that created a mess on the 469 client.  I update my UT99 451 Server to 469 and fixed that.   So now I just need to play a few rounds with some bots... but starting up and connecting to the server is working.

Is there a way for to set my UT99 to only allow 469 or above clients to connect?

**Comment by SeriousBuggie** [comment, marked as answer]
You can try change MinClientVersion in UT.ini

**Comment by Odie3** [comment]
Thank you, Infiltration MOD uses "Infiltration.ini" so I didn't see anything that looked like setting the min version. 

So it seems that for Infiltration MOD a v469 client can't connect to a v451 server (tries to send core UT files to the client).  I then went I fired up my Infiltration v436 client to connect to the updated v469 Server I was happy to see it connect but once the round started I didn't have any Infiltration weapons (actually I had them, since I could fire - I just couldn't see them or the weapons HUD).

To set the min client version from ini file:
```
[Engine.Engine]
MinClientVersion=469
```

Thanks for the help & tips - you been very helpful.  Now if I could find some old Infiltration players that would be great LOL

Edit & FYI:
I posted what I learned in the BeyondUnreal Forums under [GitHub UT99 469d for Infiltration](https://forums.beyondunreal.com/threads/github-ut99-469d-for-infiltration.209027/)

**Comment by SeriousBuggie** [comment]
Usually problem of invisible weapons caused by missed ServerPackages entries. But if it work for 469, then it must be different reason then.

**Comment by SeriousBuggie** [comment]
About splash screen - it is just BMP file in Help folder. You can restore old if want.

**Comment by Odie3** [comment]
Nice tip on the logo.  

The Infiltration.exe would rotate between a few logos, which I found them all in the ../help directory.  So it has INFINF0.bmp, INFINF1.bmp, etc.  I guess UT.exe only sees/used Logo.bmp.

Thanks!  BTW, I did donate to ya'lls project.  Thank you for all the work & help.
